test/python/transpiler/test_target.py

```diff
@@ -240,7 +240,7 @@ def test_get_qargs(self):
             self.empty_target.get_qargs("rz")
         self.assertEqual(self.ibm_target.get_qargs("rz"), {(0,), (1,), (2,), (3,), (4,)})
         self.assertEqual(self.aqt_target.get_qargs("rz"), {(0,), (1,), (2,), (3,), (4,)})
-        self.assertEqual(self.fake_backend_target.get_qargs("cx"), {(0, 1), (1, 0)})
+        self.assertEqual(self.fake_backend_target.get_qargs("cx"), {(0, 1)})
```

Comment:
How come this PR makes this necessary?

Reply:
It's not strictly necessary for this PR except that this is what I wanted to test. I wanted to test that a heterogeneous gate set with different 2q gates on different links would properly be resolved by a call to `transpile()`. This PR is basically trying to be the minimum work to ensure we're compiling valid circuits for this use case. So I changed the fake backend which was being added in #5885 to have cx only on (0, 1) to ensure that a cx isn't valid on (1, 0) in the output circuit to test this.

Reply:
Ahhh, I see - I thought the PR had actually caused the expected behaviour to change, or a test was otherwise breaking or something.  I understand.